test/unit/spec/OktaSignIn_spec.js

```diff
@@ -5,11 +5,11 @@ define([
   'util/Logger'
 ],
 function (Widget, Expect, Logger) {
-  var signIn, devLogger;
+  var signIn;
```

Comment:
Reverts change implemented in a [previous PR commit](https://github.com/okta/okta-signin-widget/pull/466/commits/7fa57dc3d16e2cdf68aae33d5d9b6be3d578a79a#r191577368)